src/Eyes/Url/UrlStatus.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace BeholderWebClient\Eyes\Url;
+
+use BeholderWebClient\Eyes\Status;
+
+class UrlStatus extends Status {
+
+  const REQUERIMENT_FAIL = 'REQUERIMENT FAIL: php lib curl is not installed.';
+
+  const COULD_NOT_CONNECT = 'Could not connect to url: ';
+  const COULD_NOT_CONNECT_NUMBER = 600;
+
+  const STATUS_CODE_WAS_NOT_EXPECTED = 'The returned status code was not the expected, received: ';
```

Comment:
it was not expected is when nothing was been expected, but it was not the expected is when something was been expected, and what happen was not it.